components/gantry/multiaxis/multiaxis.go

```diff
@@ -161,16 +161,22 @@ func (g *multiAxis) MoveToPosition(ctx context.Context, positions, speeds []floa
 }
 
 // GoToInputs moves the gantry to a goal position in the Gantry frame.
-func (g *multiAxis) GoToInputs(ctx context.Context, goal []referenceframe.Input) error {
-	if len(g.subAxes) == 0 {
-		return errors.New("no subaxes found for inputs")
-	}
-	ctx, done := g.opMgr.New(ctx)
-	defer done()
+func (g *multiAxis) GoToInputs(ctx context.Context, inputSteps ...[]referenceframe.Input) error {
+	for _, goal := range inputSteps {
+		if len(g.subAxes) == 0 {
+			return errors.New("no subaxes found for inputs")
+		}
+		ctx, done := g.opMgr.New(ctx)
+		defer done()
```

Comment:
same defer question

Reply:
This particular defer statement, I think we actually want outside of the loop, since it's the op manager